app/src/main/java/net/squanchy/venue/VenueInfoComponent.java

```diff
@@ -0,0 +1,21 @@
+package net.squanchy.venue;
+
+import android.content.Context;
+
+import net.squanchy.injection.ActivityLifecycle;
+import net.squanchy.injection.ApplicationComponent;
+import net.squanchy.navigation.NavigationModule;
+import net.squanchy.navigation.Navigator;
+
+import dagger.Component;
+
+@ActivityLifecycle
+@Component(modules = {VenueInfoModule.class, NavigationModule.class}, dependencies = ApplicationComponent.class)
+interface VenueInfoComponent {
+
+    Context context();
```

Comment:
Why do you need this here? Are you injecting the `Context` in the view? Or is there something I'm missing?